src/NLog/Internal/ExceptionHelper.cs

```diff
@@ -99,8 +100,10 @@ public static bool MustBeRethrown(this Exception exception, IInternalLoggerConte
                     InternalLogger.Log(exception, level, "Error has been raised.");
             }
 
+            logFactory = logFactory ?? LogManager.LogFactory;
```

Comment:
Think to make it non-breaking then you should both check local `logFactory` AND global `LogManager.LogFactory`

Reply:
OK, So I think

- logfactory.ThrowConfigExceptions, if null, then fallback to LogManager.ThrowConfigExceptions
- logfactory.ThrowExceptions, if false, then fallback to LogManager.ThrowExceptions? (unsure about this one)

Reply:
If one of them is `true` then it should throw and explode.

Reply:
I'm not sure if we should keep it the old behavior, is kinda bug that the properties on LogFactory aren't used.

Reply:
Yes "bug" should be fixed with NLog 5.0 where lot of static behavior is being fixed.

Reply:
Updated it!